pkg/cloudprovider/aws/instance.go

```diff
@@ -51,9 +52,13 @@ func (p *InstanceProvider) Create(ctx context.Context,
 			})
 		}
 	}
+	cancelCtx, cancel := context.WithCancel(ctx)
+	defer cancel()
 
 	// 2. Create fleet
-	createFleetOutput, err := p.ec2.CreateFleetWithContext(ctx, &ec2.CreateFleetInput{
```

Comment:
We don't want this. We want to use the ctx passed in.

Reply:
This uses the ctx passed in, but this file has it done incorrectly, sorry forgot one push.

Reply:
Should be good now.